testing/playwright/page-objects/CreateProviderPage.ts

```diff
@@ -18,66 +18,84 @@ export class CreateProviderPage {
     this.navigationHelper = new NavigationHelper(page);
   }
 
-  async create(testData: ProviderData, waitForReady = true): Promise<ProviderDetailsPage> {
-    const providerDetailsPage = new ProviderDetailsPage(this.page);
+  private async fillCredentialsFields(testData: ProviderData) {
+    if (testData.type === 'ova') {
```

Comment:
we should also consider an enum/constants for provider types since they are also widely used